server/eclipse-project/src/main/java/us/freeandfair/corla/report/CountyReport.java

```diff
@@ -283,23 +282,32 @@ public Workbook generateExcelWorkbook() {
     cell = row.createCell(cell_number++);
     cell.setCellType(CellType.STRING);
     cell.setCellStyle(bold_style);
-    cell.setCellValue("Total Ballot Cards Cast");
+    cell.setCellValue("Total Ballot Cards In Manifest");
     
     cell = row.createCell(cell_number++);
     cell.setCellType(CellType.NUMERIC);
     cell.setCellStyle(integer_style);
-    final OptionalLong ballots = 
-        CastVoteRecordQueries.countMatching(my_county.id(), RecordType.UPLOADED);
-    if (ballots.isPresent()) {
-      cell.setCellValue(ballots.getAsLong());
-    }
+    cell.setCellValue(my_cdb.ballotsInManifest());
     
     row = summary_sheet.createRow(row_number++);
     cell_number = 0;
     
     cell = row.createCell(cell_number++);
     cell.setCellType(CellType.STRING);
     cell.setCellStyle(bold_style);
+    cell.setCellValue("Total CVRs in CVR Export File");
+    
+    cell = row.createCell(cell_number++);
```

Comment:
As in the earlier PR, I suggest not using the increment operator on a variable being passed to a method, here, below, and in the other report.

Reply:
In most cases I agree, but here... no. Rewriting both reporting classes to add lines every time I increment the cell number in an Excel file would double the sizes of the classes for no real benefit (there isn't even a readability benefit, because honestly, the code required to generate the Excel files is mostly straight line "make this cell, then make this other cell, then make this row..." anyway).